src/lib/censorRecords.js

```diff
@@ -407,22 +407,22 @@ const censorBirthV1 = (birth, roles) => {
       signatureIsMark: censorField(true, birth.informant2.signatureIsMark)
     },
     child: {
-      originalPrefix: censorField(true, birth.child.originalPrefix),
-      prefix: censorField(true, birth.child.prefix),
+      originalPrefix: censorField(false, birth.child.originalPrefix),
+      prefix: censorField(false, birth.child.prefix),
       forenames: censorField(false, birth.child.forenames),
-      originalForenames: censorField(true, birth.child.originalForenames),
+      originalForenames: censorField(false, birth.child.originalForenames),
```

Comment:
As discussed elsewhere, let's redact this.

Reply:
updated

Reply:
Done